main/src/com/google/refine/util/HttpClient.java

```diff
@@ -105,6 +111,24 @@ public void process(
                     }
                 });
 
+        if (System.getProperty("http.proxyHost") != null)
+            proxyHost = System.getProperty("http.proxyHost");
+
+        if (System.getProperty("https.proxyHost") != null)
+            proxyHost = System.getProperty("https.proxyHost");
+
+        if (System.getProperty("http.proxyPort") != null)
+            proxyPort = Integer.parseInt(System.getProperty("http.proxyPort"));
+
+        if (proxyHost != null && proxyPort != 0) {
+            proxy = new HttpHost("http", proxyHost, proxyPort);
```

Comment:
Question (I have not looked through it myself): does this work for both HTTP and HTTPS proxies? Or do we need to add a configuration variable to let the user choose between the two?